docs/sdk/sdk-by-examples/simple-governance/tendermint-cosmos.md

```diff
@@ -0,0 +1,25 @@
+## Tendermint and Cosmos
```

Comment:
we have this section in the docs already - can we add this content there instead of duplicating ideas?

Reply:
Done